direct/tpm2/read_public_test.go

```diff
@@ -13,14 +13,20 @@ import (
 	"github.com/google/go-tpm/direct/transport/simulator"
 )
 
+// TestReadPublicKey compares the createPrimary PublicArea when instantiated with
+// the PublicArea read from executing readPublic.
 func TestReadPublicKey(t *testing.T) {
-
+	// Open simulated TPM for testing.
 	thetpm, err := simulator.OpenSimulator()
 	if err != nil {
 		t.Fatalf("could not connect to TPM simulator: %v", err)
 	}
+	
+	// Defer the close of the simulated TPM to after use.
```

Comment:
Without this, other programs/tests may not be able to get a handle to the TPM.

Reply:
Comment added.